src/test/java/uk/gov/pay/publicauth/utils/DatabaseTestHelper.java

```diff
@@ -78,4 +77,5 @@ public ZonedDateTime getCurrentTime() {
                         .map(new ZonedDateTimeMapper(Optional.of(TimeZone.getTimeZone("UTC"))))
                         .first());
     }
+
```

Comment:
Also if you do change the StringMapper to StringColumnMapper for the DAO above you might as well change its usage here as when you fix the unneeded params above..